virtual-organizations/BNL.yaml

```diff
@@ -30,10 +30,9 @@ ID: 114
 LongName: Brookhaven National Laboratory
 OASIS:
   Managers:
-    Jason Federico:
+    John S. De Stefano Jr.:
       DNs:
-      - /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Jason Federico 2576
-      - /DC=com/DC=DigiCert-Grid/O=Open Science Grid/OU=People/CN=Jason Federico 2576
+      -  /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=destefan/CN=665120/CN=John Steven De Stefano Jr
       ID: 9ad4d5aa3825356e2329e7fcb97c5b2fa2014b9e
```

Comment:
I'm not exactly sure how the ID is used, but judging from other files it appears your ID is 8a7d75ea65a2962520279197c72df6a16e1533a1 and it should be changed here.